src/timetable/timetable-header/timetable-header.style.tsx

```diff
@@ -39,3 +40,8 @@ export const NavigationHeaderSubtitleWrapper = styled.div`
 export const StyledResultText = styled(Text.H6)`
     color: ${Color.Neutral[3]};
 `;
+
+export const StyledDateNavigator = styled(DateNavigator)`
+    width: 400px;
+    height: 56px;
```

Comment:
remove the hardcoded height here. this lets the navigator handle multiline display (e.g. when user increases the font size)